tests/commoncode/test_fileutils.py

```diff
@@ -318,14 +318,14 @@ def test_os_walk_with_unicode_path(self):
     def test_fileutils_walk(self):
         test_dir = self.get_test_loc('fileutils/walk')
         base = self.get_test_loc('fileutils')
-        result = [(as_posixpath(t.replace(base, '')), d, f,) for t, d, f in fileutils.walk(test_dir)]
+        result = [(as_posixpath(t.replace(base, '')), d, sorted(f),) for t, d, f in fileutils.walk(test_dir)]
```

Comment:
because the `result` attribute have different values in Ubuntu and Windows : 
windows : 
```
expected = [
            ('/walk', ['d1'], ['f', 'unicode.zip']),
            ('/walk/d1', ['d2'], ['f1']),
            ('/walk/d1/d2', ['d3'], ['f2']),
            ('/walk/d1/d2/d3', [], ['f3'])
        ]
```

Ubuntu : 
```
expected = [
            ('/walk', ['d1'], ['unicode.zip', 'f']),
            ('/walk/d1', ['d2'], ['f1']),
            ('/walk/d1/d2', ['d3'], ['f2']),
            ('/walk/d1/d2/d3', [], ['f3'])
        ]
```
So, I put a `sorted()` which fixed the issue.